language/German/strings.po

```diff
@@ -2645,12 +2695,12 @@ msgstr "Hintergrundfarbe für Mausklick"
 
 #: src/frontend/mame/ui/custui.cpp:628
 msgid "Restore default colors"
-msgstr ""
+msgstr "Standardfarben wiederherstellen"
 
 #: src/frontend/mame/ui/custui.cpp:640
 #, c-format
 msgid "Double-click or press %1$s to change color"
-msgstr ""
+msgstr "Doppelklick oder Drücke %1$s um Farbe zu ändern"
```

Comment:
you use formal "Sie" everywhere, so "Doppelklicken Sie oder *drücken Sie* .... um *die* Farbe zu ändern"

Reply:
Fixed